### PR TITLE
Fix formatting and typos in CompareFlags.h

### PR DESCRIPTION
**Pull request by f0rest9999**

### CODE REVIEW

**Comment by f0rest9999** [comment]
@mbasmanova Could you please review this simple PR?

File: velox/common/base/CompareFlags.h [hunk 1]
```diff
@@ -34,113 +34,115 @@ struct CompareFlags {
...
-          - Fields are compared in order starting from index 0.
-
-          - If any two fields are different but not null, then result is
-            determined and remaining uncompared elements are ignored.
-            ex: (1, null) > (2, null) -> result is false and nulls not compared.
-
-          - If a null element is encountered before result is determined then
-            the compare throws a user exception.
-            ex: (1, null) > (1, 2) -> throws.
+    /// This is the default null handling mode, in this mode nulls are treated
```
Comment (mbasmanova) [inline velox/common/base/CompareFlags.h:38]: nit: Drop "This is" and "in this mode"

`The default null handling mode where nulls are treated...`

File: velox/common/base/CompareFlags.h [hunk 2]
```diff
@@ -34,113 +34,115 @@ struct CompareFlags {
...
+    ///    - null == null is true,
+    ///    - null == value is false.
+    ///    - when equalsOnly=false null ordering is determined using the
+    ///    nullsFirst flag.
+    kNullAsValue,
 
-        4. Maps:
-          - This mode does not allow ordering maps.
-    */
+    /// This mode implements Presto semantics for handling nulls. It matches the
```
Comment (mbasmanova) [inline velox/common/base/CompareFlags.h:46]: nit: drop 'This mode implements'

File: velox/common/base/CompareFlags.h [hunk 3]
```diff
@@ -34,113 +34,115 @@ struct CompareFlags {
...
+    /// behavior of ==, >, < functions and many other Presto functions such as
+    /// array_remove and array_contains.
+    ///
+    /// Under this mode, result of comparison can be indeterminate;
```
Comment (mbasmanova) [inline velox/common/base/CompareFlags.h:50]: ; -> period

File: velox/common/base/CompareFlags.h [hunk 4]
```diff
@@ -34,113 +34,115 @@ struct CompareFlags {
...
+    /// indeterminate result is represented as std::nullopt, and means that the
```
Comment (mbasmanova) [inline velox/common/base/CompareFlags.h:51]: Such result is represented as std::nullopt and means...

File: velox/common/base/CompareFlags.h [hunk 5]
```diff
@@ -34,113 +34,115 @@ struct CompareFlags {
...
+    /// function can not decide on the result of the comparison due to some
+    /// existing nulls. But not every null results in indeterminate result.
```
Comment (mbasmanova) [inline velox/common/base/CompareFlags.h:53]: But not every... -> Not every ...

**Review by mbasmanova — COMMENTED** [review]
Thanks. Some nits.

**Review by mbasmanova — APPROVED** [review]
Thanks.

**Pull request closed without merging**